config/manifests/bases/vault-secrets-operator.clusterserviceversion.yaml

```diff
@@ -168,4 +168,7 @@ spec:
   provider:
     name: HashiCorp
     url: https://www.hashicorp.com/
+  skips:
+  - vault-secrets-operator.v0.5.0
+  - vault-secrets-operator.v0.5.1
```

Comment:
Because of these I think we'll want to use PREVIOUS_VERSION=v0.4.3 with `make bundle`:

```
make bundle VERSION=${LATEST_TAG//v} USE_IMAGE_DIGESTS=true PREVIOUS_VERSION=v0.4.3
```

So that the resulting `build/bundle/manifests/vault-secrets-operator.clusterserviceversion.yaml` comes out like this:

```yaml
  skips:
    - vault-secrets-operator.v0.5.0
    - vault-secrets-operator.v0.5.1
  version: 0.5.2
  replaces: vault-secrets-operator.v0.4.3
```